Bottom: 40296b471bb0f3c6843f617b24e44d2ba56b1cee
Top:    4d2a9ae39c62a8c793af91f810421b06e4868822
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-05 16:42:49 +0800

Refresh of virtio-net-basic-ebpf-offload

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index cb8a85f..1cebf61 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -224,6 +224,9 @@ struct virtnet_info {
 
 	struct dentry *ddir;
 
+	struct bpf_prog *bpf_offloaded;
+	u32 bpf_offloaded_id;
+
 	bool bpf_bind_accept;
 	u32 bpf_bind_verifier_delay;
 	struct dentry *ddir_bpf_bound_progs;
@@ -2396,7 +2399,7 @@ static int virtnet_bpf_init(struct virtnet_info *vi)
 	debugfs_create_u32("bpf_offloaded_id", 0400, vi->ddir,
 			   &vi->bpf_offloaded_id);
 
-	ns->bpf_bind_accept = true;
+	vi->bpf_bind_accept = true;
 	debugfs_create_bool("bpf_bind_accept", 0600, vi->ddir,
 			    &vi->bpf_bind_accept);
 	debugfs_create_u32("bpf_bind_verifier_delay", 0600, vi->ddir,
